### ISSUE TITLE
[BUG] `nvmd use {version} --project` fails with `nvm-desktop: The system cannot find the file specified. (os error 2)`

### ISSUE THREAD

**Issue by iki** [1111mp/nvm-desktop#130]
**Describe the bug**

Running `nvmd use {version} --project` in a project directory fails with `nvm-desktop: The system cannot find the file specified. (os error 2)`

Running global `nvmd use {version}` and node/npm/npx/corepack shims works OK

**To Reproduce**

1. Install `NVM.Desktop_4.0.1_x64-setup.exe` on Windows 11 x64
2. Install Node v22.9.0 via NVM Desktop
3. **[OK]** Optionally check `%UserProfile%\.nvmd\bin` is in PATH
4. **[OK]** Optionally check Node v22.9.0 is installed in `%UserProfile%\.nvmd\versions\22.9.0`
5. Set default Node version (see #128)
```sh
$ nvmd use v22.9.0
Now using node v22.9.0
```
6. **[OK]** Try running node/npm/npx/corepack via nvmd shim. Here I use explicit path to be sure:
```sh
$ "%UserProfile%\.nvmd\bin\node.exe" --version
```
7. **[BUG]** Try setting Node version on a project
```sh
$ cd "{project directory with package.json}"
$ nvmd use v22.9.0 --project
nvm-desktop: The system cannot find the file specified. (os error 2)
```

**Expected behavior**
- Setting Node version on a project should work OK
- Error message should include path of the file that cannot be found

**Desktop (please complete the following information):**
 - OS: Windows 11 23H2 Home 10.0.22635.4367
 - Arch: x64
 - NVM Desktop Version: 4.0.1

**Comment by 1111mp** [comment]
This issue has been fixed in [Release v4.0.2](https://github.com/1111mp/nvm-desktop/releases/tag/v4.0.2). You can upgrade to this version to verify it. Thanks. 🌹

**Comment by iki** [comment]
Works perfectly! Many thanks!
